backends-clickhouse/src/main/scala/org/apache/gluten/backendsapi/clickhouse/CHIteratorApi.scala

```diff
@@ -122,7 +122,8 @@ class CHIteratorApi extends IteratorApi with Logging with LogLevelUtil {
       partition: InputPartition,
       partitionSchema: StructType,
       fileFormat: ReadFileFormat,
-      metadataColumnNames: Seq[String]): SplitInfo = {
+      metadataColumnNames: Seq[String],
+      scan: BasicScanExecTransformer): SplitInfo = {
```

Comment:
It is better to only pass the properties that are needed.

Reply:
The other parameters of this method are also from the scan transformer. It seems a bit strange to directly pass BasicScanExecTransformer here. I think it would be better to add additional parameters for other fixes.

Reply:
Let me fix it.